### DISCUSSION TITLE
Introduce Yourself!

### DISCUSSION THREAD

**Discussion by Erin-Cecele** [thegetty/quire#133] [General]
Hello, and welcome to the Quire community!

Getty enabled beta access for Quire in early 2018, and our community of users has been steadily growing ever since.  As we continue developing and expanding Quire, working towards an open-source, community-supported model, we'd love for everyone to get to know one another. The most important element of a thriving community is engagement and participation, so we invite you to introduce yourself to your fellow digital publishing colleagues:

- Tell us about yourself? We'd love to hear more about who you are, what you do, and where you are based. 
- What excites you the most about Quire as a publishing tool?
- What's your favorite thing about the Quire community?
- What are some of your interests and passions outside of digital publishing?

We look forward to hearing from all of you and building a vibrant community together!

**Comment by rabarth1** [comment]
Hey everyone! My name is Rachel and I'm an associate editor in Getty Publications, Los Angeles. I've worked on a handful of Quire books now...maybe almost 10, ranging from small publications to large collections catalogues. I love that with Quire, we can have brainstorming conversations about adding new elements to a book that wouldn't be possible if the book was print only. I love that a Quire book on ancient Greek or Roman art lets you zoom in such a way that you can see the cracks and fingerprints in the object. I don't know too much about the Quire community outside the Getty yet, but I am so glad that people are using it and experimenting with it. My hope is that museums and other arts organizations can find ways to make publishing their books on Quire in a cost-effective way, because making this material free and available to the public is so important!

**Comment by dimrkh** [comment]
👋 !! I'm Dina 🤓 . I'm an art history PhD candidate and a graduate intern in the Publications department of the Getty in LA. My dissertation explores the way people have historically engaged with the same content in different media such as paintings, lithographs, drawings, books, newspapers, advertising, letters, personal albums, and more, so I find Quire's promise of easy transmission across website, PDF, and print formats to be a cool contemporary case study in this kind of intermedia experience. I've become increasingly familiar with Quire over the last few months, and have enjoyed experimenting with the minutiae of formatting as well as seeing the bigger picture of a whole book come together. Quire's proximity to the idea of the printed book while also offering new solutions for high-resolution imaging and easy hyperlinking to resources and references seems to be a great step towards accessibility and longevity. I've been fascinated by the different uses for Quire that the community has found, including catalogues, journals, class projects, and even family history. Outside of work, I love 🎶 🎙 🎾 🏓 📚 !

**Comment by aebancroft** [comment]
Hello! 🙋‍♀️ My name is Alexandra Bancroft and I'm the Technical Support Specialist in the Getty Museum's Collection Information & Access department. We started using Quire a couple years ago to publish curator's scholarly essays on our website's collection pages. We were so thrilled to find we could use Quire for this ongoing project. We had a few goals we were trying to meet simultaneously:

1. Fit essays into current collection page design
2. Maintainable by multiple people sometimes w/ varying levels of technical expertise
3. Establish workable publishing workflow between curators and Collection Info & Access
4. Include comparable images with captions and credit lines

Thanks to Quire's flexibility as a full-service digital publishing tool we're able to meet all of these goals. With the help of our technical partners at Getty we were able to add the Quire framework to our collection pages. The essays can also stand alone as separate webpages. Here is an example of the [collection page for Gauguin's _Arii Matamoe (The Royal End)_ ](http://www.getty.edu/art/collection/objects/243174/)(see essay tab at bottom of page). And here is a direct link to the [essay's full page](https://museum-essays.getty.edu/paintings/sallan-gauguin/). During the publication process we're able to provide curators and registrars with staged versions of the essay for final approvals before pushing live to our website. 

I've enjoyed seeing our project take shape and learning how to use Quire. The [Get Help](https://github.com/thegetty/quire/discussions/categories/get-help) section and Quire community have been so helpful when I've gotten stuck. It's great to see how others are using Quire and I look forward to continuing to engage with this innovative community! 😄

**Comment by gspy72** [comment]
Hello! I am Georgios Spyropoulos, one of the authors of "Egypt and the
Classical World: Cross-Cultural Encounters in Antiquity" It has been a
great pleasure working with Quire. I work closely with all my Getty
colleagues, especially Jeffrey Spier and Sara Cole, and I thank you all for
being so immensely helpful and supportive on all fronts.

Withe renewed thanks and best wishes,

Georgios Spyropoulos.


Prof. Dr  Georgios Spyropoulos.
Academic Associate, Ioannou Center for
Classical and Byzantine Studies,
University of Oxford

**Comment by rrolonNelson** [comment]
Howdy! 
My name is Rachel Rolon I am a Developer at the Nelson Atkins Museum of Art in Kansas City.   I went to art school to be a textile artist but ended up being a programmer!  I have only been at the museum for a few months now, so I am still finding my way around everything.  
I am excited to be involved in two catalog projects and their digital release to the public.  I look forward to the direction Quire is going and really appreciate the assistance the Getty team has provided! (it is also really neat to see how this project has brought together several institutions and people).  
Outside of digital publishing I am passionate about open source projects and urban farming.

**Comment by leslie-martinez15** [comment]
Hi! My name is Leslie Martinez and I am a current Getty Marrow Intern working in the Getty Publications for this summer. I have worked on creating alt-texts for a catalog and am currently working on my own Quire project! I have always been really intimidated by the idea of web design and just working anything relatively with coding. However, Quire has helped open that door and learning the functions of Quire has been lots of fun! What I think is really neat is that once you learn the basics you can explore and find other ways you can make your Quire function/look. Which explains the different variety of Quire entries there is found in the community showcase; you have the opportunity to share information in a fun way! Quire has been a comfortable entry way on learning the new horizons of digital publishing.

**Comment by aheltonsjma** [comment]
Hi there! I worked on a Quire project which launched in 2020: https://50x50.sjmusart.org/

I am on my way out of the museum actually but still very much in love with QUIRE! 
Please be in touch if you want to hear me wax poetic about this wonderful platform or learn more about my project/setting up your own! 

After March 12th (my last day at SJMA) I can be reached here: ahelton897@gmail.com, 8653608145 

Please reach out! Happy publishing! 
Amanda
